applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_3D/3D_contact_simplest_patch_matching_adaptative_test_parameters.json

```diff
@@ -27,7 +27,7 @@
             "max_number_splits"              : 1
```

Comment:
Do these changes belong to this PR?

Reply:
Yes, it was a test failing in the contact, the deformation imposed is so big that it fails with this change. Reducing time step works (because the displacement is imoosed few by few)